_includes/repository/repo_user.liquid

```diff
@@ -1,3 +1,23 @@
+{% assign lang = site.active_lang | split: '-' | first %}
+
+{% case lang %}
+  {% when 'pt' %}
+    {% assign lang = site.active_lang %}
+
+  {% when 'zh' %}
+    {% assign lang_last = site.active_lang | split: '-' | last %}
+    {% case lang_last %}
+      {% when 'cn', 'sg', 'my', 'hans' %}
+        {% assign lang = 'cn' %}
+      {% when 'tw', 'hk', 'mo', 'hant' %}
+        {% assign lang = 'zh-tw' %}
+
+        {% comment %} when... add a new language here if needed {% endcomment %}
```

Comment:
This comment must be after the `endcase`, if not the language would be added inside the `zh` case.

Reply:
Can you move the comments to inside the outer `case`? Since there is where the `when` must be added, so it is clear to the user.